data/helpers.d/setting

```diff
@@ -176,8 +207,8 @@ else:
     elif action == "set":
         if key in ['redirected_urls', 'redirected_regex']:
             value = yaml.load(value)
-        if any(key.startswith(word+"_") for word in ["unprotected", "protected", "skipped"]):
-            sys.stderr.write("/!\\ Packagers! This app is still using the skipped/protected/unprotected_uris/regex settings which are now obsolete and deprecated... Instead, you should use the new helpers 'ynh_permission_{create,urls,update,delete}' and the 'visitors' group to initialize the public/private access. Check out the documentation at the bottom of yunohost.org/groups_and_permissions to learn how to use the new permission mechanism.\n")
+        if any(key.startswith(word+"_") for word in ["unprotected", "protected", "skipped", "noauth"]):
+            sys.stderr.write("/!\\ Packagers! This app is still using the skipped/protected/unprotected_uris/regex/noauth settings which are now obsolete and deprecated... Instead, you should use the new helpers 'ynh_permission_{create,urls,update,delete}' and the 'visitors' group to initialize the public/private access. Check out the documentation at the bottom of yunohost.org/groups_and_permissions to learn how to use the new permission mechanism.\n")
```

Comment:
Eh, I don't think this is needed nor desirable to have this deprecation warning for "noauth" since it's not replaced by something in the new permission system ...? (same thing happening in app.py)

Reply:
Thanks for the help, my bad.. I still have some issue to read and understand code....